sema/commons/web/conneg.py

```
@@ -45,7 +45,7 @@ def add_variant(
             else None
         )
         cdispval, cdispparams = cgi.parse_header(
-            response.headers.get("Content-Disposition", "")
+            response.headers.get("Content-Disposition", "")  # type: ignore
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)